### PR TITLE
Finalize documentation

### PR DESCRIPTION
**Pull request by jhchee18**
- [x] UG
- [x] DG (include test cases)
- [x] PPP

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -988,7 +989,45 @@ testers are expected to do more *exploratory* testing.
...
+       Expected: Interview failed to be added. Error details are shown.
+
+### Removing an interview
+
+1. Removing an interview from an application.
+    1. Test case: `remove-i 1`<br>
+       Expected: Interview with displayed index 1 is successfully removed from its application, success message with interview details is shown.
+
+    2. Test case: `remove-i 0`<br>
+       Expected: Interview failed to be added. Error details are shown.
```
Comment (jolynloh) [inline docs/DeveloperGuide.md]:
```suggestion
       Expected: Interview failed to be removed. Error details are shown.
```

**Review by jolynloh — COMMENTED** [review]
LGTM!

**Comment by jhchee18** [comment]
the html somehow cannot display properly on the website, i modified it to bullet list instead.

Before: (website)
image

After: (preview)
image

**Comment by teekaytai** [comment]
> the html somehow cannot display properly on the website, i modified it to bullet list instead.

Thanks for spotting that! Yup we definitely gotta look through the websites a last time once everything's merged.

**Pull request merged**